weave/trace/weave_client.py

```diff
@@ -1181,11 +1177,11 @@ def _add_score(
 
         freq = FeedbackCreateReq(
             project_id=self._project_id(),
-            weave_ref=call_ref_uri,
-            feedback_type="wandb.runnable." + score_name,
+            weave_ref=weave_ref_uri,
+            feedback_type="wandb.runnable." + runnable_ref.name,
             payload=payload,
-            runnable_ref=scorer_op_ref_uri,
-            call_ref=scorer_call_ref_uri,
+            runnable_ref=runnable_ref_uri,
+            call_ref=call_ref_uri,
```

Comment:
these fields represent URIs so we should call them uris

Reply:
changing this is outside the scope of this PR - it is the interface layer

Reply:
are you referring to the variable or the req field?

Reply:
The field -- it's called `ref` but it's being assigned a `uri`.  I think it actually represents a URI?

Reply:
yes, but that follows the existing pattern of the feedback table and the interface. Changing this would impact quite a few layers and break pattern

Reply:
ok, unfortunate but I wont block here.

There are a few places where the nomenclature is kinda weird and I always get a little confused when I revisit them